src/Part2.js

```diff
@@ -0,0 +1,133 @@
+import axios from "axios";
+
+
+const sectionCountries = document.getElementById('country-information');
+let errorMessage = "";
+
+
+async function fetchDataSearch(countryName) {
+
+    try {
+
+        const dataCountry = await axios.get(`https://restcountries.com/v2/name/${countryName}`);
+
+
+        // Leeg de lijst
+        sectionCountries.replaceChildren();
+        errorMessage.replaceChildren();
+
+
+
+        // Map door de data heen
+        dataCountry.data.map( ( country ) => {
+
+            //maak variable aan uit arrays
+            const {flags:{png}, name, capital, subregion, currencies, population, languages} = country;
+
+
+            //make array list of needed names.
+            let coinString;
+            const coinArray = currencies.map((coin) => {
+                return coin.name;
+            })
```

Comment:
Hier is `coinArray` niets anders dan `currencies`, dus het maken van de array is overbodig